english/total-war/war-start.md

```diff
@@ -0,0 +1,10 @@
+Total War - Warhammer is a turn-based strategy game with 
+epic real-time tactical battles between armies.
+
+It is also the first in its franchise that introduces a 
+fantasy setting from the [Warhammer universe](total-war/universe/universe.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.